utils/api.ts

```diff
@@ -12,192 +13,12 @@ export const getApi = () => {
   return api
 }
 
-export const initApi = async (uri = '') => {
-  // const wsProvider = new WsProvider(process.env.API_URL || 'ws://mangata-node:9944')
+export const initApi = async ( uri = '') => {
   if(!uri)
-    uri = envUri;
+  uri = envUri;
```

Comment:
this looks weird everything is on the same line.. you have there if statement. I generally do not like if statement without brackets
```
if (!uri) {
  uri = envUri
}
```

Reply:
so.. are you suggesting removing traces or something else? sorry I don't fully understand the comment :)

Reply:
think i fixed. missing tab :)

Reply:
exactly missing tab ;)